src/FtxExchangeConfiguration.ts

```diff
@@ -44,6 +47,45 @@ export class FtxExchangeConfiguration implements ExchangeConfiguration {
     }
   }
 
+  fetchDepositsValidateInput(address: string, amountInSats: number) {
+    assert(address, ApiError.UNSUPPORTED_ADDRESS)
+    assert(amountInSats, ApiError.NON_POSITIVE_QUANTITY)
+    assert(amountInSats > 0, ApiError.NON_POSITIVE_QUANTITY)
+  }
```

Comment:
did you enable the linter? there is no space between the 2 functions for some reasons

Reply:
yes I did and run it before commits, it seems to accept 0 or 1, but removes extra if any.